app/views/moderations/mod.html.erb

```diff
@@ -155,6 +156,46 @@
       <% end %>
     </div>
   <% end %>
+
+  <% if @moderatable.class.name == "Article" && (current_user.has_role?(:super_admin) || current_user.has_role?(:tag_moderator, :any)) %>
```

Comment:
this too could become a variable in the controller's action. and here you can check it that is true